### PR TITLE
Metabase dump/init file and some makefile optimzations

### PR DESCRIPTION
**Pull request by woolfg**
- adds db export functionality to create metabase dump
- adds metabase init sql file
- adds target to copy sample env files
- optimized make help to find targets with spaces
- readme extended

fix openpodcast/roadmap#91

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -15,6 +15,10 @@ up-all: ## Start the docker containers (including forwarder)
 	docker-compose --profile all up
 	docker-compose logs -f
 
+.PHONY: env 
+env: ## Copy alls env sample files to .env files
```
Comment (mre) [inline Makefile]:
```suggestion
env: ## Copy all env sample files to .env files
```

File: Makefile [hunk 2]
```diff
@@ -15,6 +15,10 @@ up-all: ## Start the docker containers (including forwarder)
...
+	for file in $$(find . -name ".env.*.sample"); do cp $${file} $${file%.*}; done
```
Comment (mre) [inline Makefile:20]: I don't understand what this is useful for. 🤔

File: Makefile [hunk 3]
```diff
@@ -48,3 +56,9 @@ logs: ## Show logs
 send-api-req-local: ## Send a request to the local running server
 	curl -X POST http://localhost:8080/events -H 'Content-Type: application/json' -H 'Authorization: Bearer cn389ncoiwuencr' --data-binary "@./fixtures/forwarder.json" 
 
+.PHONY:dumpmetabase
+dumpmetabase: ## dump the database from db container
+	docker-compose exec db /bin/sh \
+	-c 'mysqldump -uroot -proot --databases metabase' \
+	| grep -v "Using a password on the command line interface can be insecure" \
```
Comment (mre) [inline Makefile:63]: A little hacky. Apparently an alternative is to set an env var with the password:
https://stackoverflow.com/a/24188878

Not sure if it works with `mysqldump`.

```
MYSQL_PWD=root mysqldump -uroot --databases metabase
```

File: Makefile (hunk 2 above)
Comment (mre) [reply Makefile:20]: Okay, the README made it a little clearer. Maybe we should change the help message. How about "Create required .env files from sample files"?

File: docker-compose.yml [hunk 4]
```diff
@@ -54,6 +54,7 @@ services:
     volumes:
       - ./db_schema/schema.sql:/docker-entrypoint-initdb.d/schema.sql
       - ./db_schema/metabase.sql:/docker-entrypoint-initdb.d/metabase.sql
```
Comment (mre) [inline docker-compose.yml]: Maybe we should rename this file to make it clearer that this is our custom config.
E.g. `metabase_custom.sql` or `metabase_admin.sql`.

**Review by mre — APPROVED** [review]

File: Makefile (hunk 2 above)
Comment (woolfg) [reply Makefile:20]: changed

File: Makefile (hunk 3 above)
Comment (woolfg) [reply Makefile:63]: found that as well but didn't work as expected.

File: docker-compose.yml (hunk 4 above)
Comment (woolfg) [reply docker-compose.yml]: changed it to metabase_user

**Pull request merged**